addon/props/model_props.py

```diff
@@ -86,6 +86,21 @@ class SOURCEOPS_ModelProps(bpy.types.PropertyGroup):
         default=False,
     )
 
+    transform_source: bpy.props.EnumProperty(
+        name='Transform Source',
+        description='Method of specifying $origin and $scale.\nEither manually specified in this panel, or via an object',
```

Comment:
I was going to mention that it also affects rotation, but it is I who forgot that $origin includes rotation, so uh ignore this comment.